install_script.sh.template

```diff
@@ -549,9 +588,16 @@ function install_installer() {
     (_install_installer "$sudo_cmd" "$os" "$distribution" > /tmp/datadog-installer-stdout.log 2> /tmp/datadog-installer-stderr.log) || exit_status=$?
     exit_status=${exit_status:-0}
 
+    local packages_to_install
+    local pacakges_to_install_after_installer
+
+    packages_to_install=("${!pkg_array_name}")
+    filter_packages_installed_by_installer "$sudo_cmd" "$pkg_array_name"
+    packages_to_install_after_installer=("${!pkg_array_name}")
```

Comment:
Is this only computed for telemetry or is it somehow propagated outside of this function?

Reply:
only used for telemetry